pubspec.yaml

```diff
@@ -1,12 +1,12 @@
 name: catcher
 description: Plugin for error catching which provides multiple handlers for dealing with errors when they are not caught by the developer.
-version: 0.6.9
+version: 0.7.0
 #author: Jakub Homlala <jhomlala@gmail.com>
 homepage: https://github.com/jhomlala/catcher
 
 environment:
-  sdk: '>=2.12.0 <3.0.0'
-  flutter: ">=1.12.0"
+  sdk: '>=2.17.0 <3.0.0'
```

Comment:
This causes issues on older projects. Can this not be reverted back?

`The current Dart SDK version is 2.16.1.
Because xyz depends on catcher from git which requires SDK version >=2.17.0 <3.0.0, version solving failed.`

Reply:
This PR is meant to discontinue older Flutter versions and fix bugs in Flutter 3 instead.
Hybrid approaches are not something I would personally recommend, as stated above.
If you need to stay on Flutter 2 (for some reason), just continue using catcher 0.6.9.

Reply:
Is there any benefit of not updating to the newest SDK version?
Or is there any incompatibilities?

Reply:
Our project, and other packages use `>=2.12.0 <3.0.0`. This appears to be the gold standard on all official Flutter packages, as its flexible enough to be used in older solutions and brand new ones.

Since this pull request has been changed too `>=2.17.0 <3.0.0` it has made it incompatible as I have mentioned above.

The benefit of reverting this back is that Catcher can be used in a lot more projects than would be possible as is.

Reply:
But this is exactly my question:
In your case, my fork of catcher is not really the problem, but rather another package that is restricting the SDK version to something like `>=2.12.1 <2.17`, right?
Maybe I just don't understand the problem correctly, but normally my fork of catcher should be compatible with projects that use `>= 2.12.1 <3.0.0` since 2.17.x is entirely within these bounds. So, no package conflicts may arise from this.
Maybe I just completely miss the point here, but that is my current understanding of the situation